src/app/checklists/items-list/item/item.component.ts

```diff
@@ -32,6 +32,13 @@ export class ChecklistItemComponent {
 
   readonly ChecklistItem_Type = ChecklistItem_Type;
 
+  onEdit(e: Event) {
+    e.stopPropagation();
+    this.promptInput?.edit();
```

Comment:
Can probably use ! for those - I don't see how they wouldn't be set at this point.

Reply:
Ah, I see, so if they can be missing in general, you declare them as `?`, but in places where they logically must be present, you use `!` to denote that understanding, and have it throw you if you made a logical error. Thanks for cleaning this up!

Reply:
Yup.
`?` translates to `if (this.promptInput) { this.promptInput.edit(); }`
`!` translates to just `this.promptInput.edit();` (i.e. assuming it won't be null)